metaci/release/utils.py

```diff
@@ -83,10 +83,101 @@ def send_release_webhook(project_config, release):
     )
     result = response.json()
     if result["success"]:
+        #######################################################################
+        # PARSE RESULT HERE FOR IMPLMENTATION ID's when GUS SANDBOX is ready! #
+        #######################################################################
         with transaction.atomic():
+            release.implementation_steps.filter(plan__role="push_sandbox").update(
+                implementation_step_id="12345"
+            )
+            release.implementation_steps.filter(plan__role="push_production").update(
+                implementation_step_id="54321111"
+            )
             case_id = result["id"]
             case_url = settings.METACI_CHANGE_CASE_URL_TEMPLATE.format(case_id=case_id)
             release.change_case_link = case_url
             release.save()
     else:
         raise Exception("\n".join(err["message"] for err in result["errors"]))
+
+
+def send_start_webhook(project_config, release, role):
+    if release is None or not settings.METACI_RELEASE_WEBHOOK_URL:
+        return
+    logger.info(
+        f"Sending start webhook for {release} to {settings.METACI_RELEASE_WEBHOOK_URL}"
+    )
+    if role == "push_sandbox":
+        implementation_step_id = release.implementation_steps.get(
+            plan__role="push_sandbox"
+        ).implementation_step_id
+    if role == "push_production":
+        implementation_step_id = release.implementation_steps.get(
+            plan__role="push_production"
+        ).implementation_step_id
```

Comment:
Instead of looking specifically for these roles, you could query for implementation steps with a matching role:

    implementation_step_id = release.implementation_steps.get(plan__role=role)